.cirrus.yml

```diff
@@ -5,8 +5,9 @@ env:  # Global defaults
   TEST_RUNNER_PORT_MIN: "14000"  # Must be larger than 12321, which is used for the http cache. See https://cirrus-ci.org/guide/writing-tasks/#http-cache
   CI_FAILFAST_TEST_LEAVE_DANGLING: "1"  # Cirrus CI does not care about dangling process and setting this variable avoids killing the CI script itself on error
   CCACHE_SIZE: "200M"
-  CCACHE_DIR: "/tmp/ccache_dir"
+  CCACHE_DIR: '${CIRRUS_WORKING_DIR}/ccache_dir'
```

Comment:
This will still be in the same volume (tmp), so why is this changed?

Reply:
I think the difference is not using absolute path `/tmp/...`.

Reply:
I am still not sure what this means and which paths are preferred or discouraged

Reply:
https://github.com/bitcoin/bitcoin/pull/27084#issuecomment-1438041890